src/js/components/Meter/stories/Labelled.js

```diff
@@ -4,26 +4,30 @@ import { storiesOf } from '@storybook/react';
 import { Grommet, Box, Meter, Stack, Text } from 'grommet';
 import { grommet } from 'grommet/themes';
 
-const LabelledMeter = () => (
-  <Grommet theme={grommet}>
-    <Box align="center" pad="large">
-      <Stack anchor="center">
-        <Meter
-          type="circle"
-          background="light-2"
-          values={[{ value: 30 }]}
-          size="xsmall"
-          thickness="small"
-        />
-        <Box direction="row" align="center" pad={{ bottom: 'xsmall' }}>
-          <Text size="xlarge" weight="bold">
-            30
-          </Text>
-          <Text size="small">%</Text>
-        </Box>
-      </Stack>
-    </Box>
-  </Grommet>
-);
+function LabelledMeter() {
+  const meterValue = 30;
+
+  return (
+    <Grommet theme={grommet}>
+      <Box align="center" pad="large">
+        <Stack anchor="center">
+          <Meter
+            type="circle"
+            background="light-2"
+            values={[{ value: meterValue }]}
+            size="xsmall"
+            thickness="small"
+          />
+          <Box direction="row" align="center" pad={{ bottom: 'xsmall' }}>
```

Comment:
You could store `{ bottom: 'xsmall' }` to avoid unnecessary re-render.

Reply:
Related to your other replies about these causing re-renders, I'm not sure I understand.

As far as I can see, the difference between storing that in a variable outside the render function, and declaring it as an object inline, is with your approach you aren't creating a new object instance when the component re-renders.

However, that component would only be re-rendered if either
1. The parent got re rendered and the child had no shouldComponentUpdate, in which case it's irrelevant what prop we pass, it's getting re-rendered either way
2. The box component had internal state change forcing a re-render, again, the props being passed are not relevant.

Please correct me if I'm missing the point, my whole team does this a lot at work so I'm interested to know if there is some performance issue here, but if there is I cannot see it

Reply:
There is no performance issue here and it won't cause any unnecessary re-renders. the prop doesn't change and assinging this to a variable doesn't affect this.